### PR TITLE
Add Abel-Ruffini done topologically

### PR DESCRIPTION
**Pull request by Incompleteusern**
This proof originates from Arnold (1953), and this is a write-up of it. It's put after complex analysis with holomorphic square roots, but only really depends on very little from the previous function and some knowledge of group theory. Found this proof from https://www.youtube.com/watch?v=BSHv9Elk1MU and took some expositional ideas.

First time actually writing new napkin content, so editorial changes/feedback are welcome and appreciated. For convenience there's a pdf of the complex analysis chapters + this new chapter attached.
[Napkin.pdf](https://github.com/vEnhance/napkin/files/13261012/Napkin.pdf)

### CODE REVIEW

File: tex/complex-ana/quintic.tex [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+\end{remark}
+
+\section{Step 1: The Simplest Case}
+
+Let's first ignore the $\sqrt[n]{}$ operator for motivation.
+Suppose I told you that some rational function $R$ always finds 
+a root of a quintic polynomial $P(z) = (z - z_1)(z - z_2)(z - z_3)(z - z_4)(z - z_5)$.
+For simplicity, let all the roots be distinct.
+
+Suppose its initially equal to $z_1$. Now, suppose we smoothly swap the roots $z_1$ and $z_2$
```
Comment (user202729) [inline tex/complex-ana/quintic.tex:39]: typo "it's"

File: tex/complex-ana/quintic.tex [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+Since $R$ is continuous, it must be tracking the same root. However, once we finish 
+swapping $z_1$ and $z_2$, the coefficients of $P$ are the same as they were initially.
+But this means that $R$ has been tricked into changing the root it outputs, contradiction!
+
+The bigger picture here is that we were able to find an operation that fixes $R$ while changing the order 
+of the roots in $S_5$.
+
+\section{Step 2: Nested Roots}
+
+Once we add $\sqrt[n]{}$ back to the picture, this idea no doesnlonger works out of the box.
```
Comment (user202729) [inline tex/complex-ana/quintic.tex:77]: more typo here...

File: tex/complex-ana/quintic.tex [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    \]
+    give $z_1$ and $z_2$.
+
+    So why can't swapping $z_1$ and $z_2$ yield a contradiction here?
+    It's because while all the coefficients end up in the starting position,
+    the liftings of how $\sqrt{b^2 - 4ac}$ travels may not.
+\end{example}
+
+\begin{exercise}
+    Consider the polynomial $z^2 - 1$. Then smoothily swap the roots along
```
Comment (user202729) [inline tex/complex-ana/quintic.tex:93]: more more typo

File: tex/complex-ana/quintic.tex [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+Let's now consider the next simplest case of the $n$th root of a rational function
+$\sqrt[n]{R}$, and try to fix it with a nontrivial permutation of the roots.
+
+Swapping the roots $z_1$ and $z_2$, we keep $R$ the same, but $R$'s path $\alpha$ around 
+the origin may have accumulated some change in phase $2\pi a$. If we were to unswap 
+$z_1$ and $z_2$ in the same manner, we'd undo the change in phase, but we'd also be back
+to doing nothing.
+
+However, while changes in phase are \emph{abelian}, while permutations are not. Let's consider
```
Comment (user202729) [inline tex/complex-ana/quintic.tex:106]: grammar error

File: tex/complex-ana/quintic.tex [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+we can now show that there is no general formula for 
+the roots of a quintic polynomial.
+
+\section{The Game Plan}
+
+Firstly, what do we even mean by ``formula'' here?
+
+\begin{definition}
+    A \vocab{quintic formula} would be a formula taking in the coefficients $(a, \dots, f)$ of a 
+    degree $5$ polynomial $P$, using the operations $+, -, \times, \div, \sqrt[n]{}$
```
Comment (user202729) [inline tex/complex-ana/quintic.tex:14]: the typesetted output doesn't look quite right. Maybe try adding a `\cdot` inside?

File: tex/complex-ana/quintic.tex [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+
+So all that remains is showing that $S_5$ is not solvable.
+We leave the housekeeping as an exercise.
+
+\section{Summary}
+
+While this is indeed a valid proof, it has some pros and cons. 
+As a con, we haven't shown that any specific quintic polynomial 
+has no solvable root (like the standard example $z^5 - z - 1$), only that 
+it can't be consistent.
```
Comment (user202729) [inline tex/complex-ana/quintic.tex:177]: need some rephrasing.

File: tex/complex-ana/quintic.tex [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+
+\begin{example}[Cubic Formula]
+    The cubic formula contains a nasty term
+    \[
+        \sqrt[3]{\frac{2b^3 - 9abc + 27a^2d + \sqrt{(2b^3 - 9abc + 27a^d)^2 - 4(b^3 - 3ac)^3}}{2}}.
+    \]
+    Here, we've taken multiple roots.
+\end{example}
+
+With our new tools, we can again consider their \emph{commutators}, which
```
Comment (user202729) [inline tex/complex-ana/quintic.tex:145]: I can roughly understand the commutator part (after squinting for a bit what it's meant by Abelian), but it's not clear how the generalization is done.

**Review by user202729 — COMMENTED** [review]
I'm not Evan, so just some comments.

File: tex/complex-ana/quintic.tex (hunk 7 above)
Comment (Incompleteusern) [reply tex/complex-ana/quintic.tex:145]: I'll try to reword this, but essentially we have operations that fix radicals of rational functions, so we can again take their commutator and the phase cancels out. Maybe I should mention explicitly that these are commutators of commutators.

There's also some exposition at https://web.williams.edu/Mathematics/lg5/ArnoldQuintic.pdf that might make more sense.

File: tex/complex-ana/quintic.tex (hunk 5 above)
Comment (Incompleteusern) [reply tex/complex-ana/quintic.tex:14]: I've done that.

File: tex/complex-ana/quintic.tex (hunk 6 above)
Comment (Incompleteusern) [reply tex/complex-ana/quintic.tex]: I have attempted to rephrase this.

File: tex/complex-ana/quintic.tex (hunk 7 above)
Comment (Incompleteusern) [reply tex/complex-ana/quintic.tex]: I went and rephrased this as well.

File: tex/complex-ana/quintic.tex [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+	using the operations $+$, $-$, $\times$, $\div$, $\sqrt[n]{}$
+	finitely many times, that maps to the five roots $(z_1, \cdots, z_5)$ of $P$.
+\end{definition}
+
+Now, any proposed quintic formula $F$ receives the same coefficients when the roots are the same,
+and thus gives the same output. This is fine at first glance, but swapping two roots continuously
+might pose more issues. $F$ must create and preserve the order of the roots under these permutations.
+
+\begin{ques}
+	Convince yourself this is true.
```
Comment (vEnhance) [inline tex/complex-ana/quintic.tex:22]: What does "this" refer to here?

**Review by vEnhance — CHANGES_REQUESTED** [review]
You should get an editor that shows trailing whitespace ;) see 6ecd895

The commit cdf77ac introduced a lot of bad changes (like commenting out all the non-complex analysis chapters) that I'm pretty sure were unintentional. So they got reverted in 9faf96f. Please glance at that and make sure everything is in good order.

Math-wise, this looks good. I did some minor copy edits but no substantial changes.

We also need to update the chart at the beginning of the textbook with the chapter numbers if we're adding a new chapter like this. Do you want to do so?

File: tex/complex-ana/quintic.tex (hunk 8 above)
Comment (Incompleteusern) [reply tex/complex-ana/quintic.tex:22]: That $F$ basically "tracks" the order of the roots as you swap them along a path.

Comment (vEnhance) [reply tex/complex-ana/quintic.tex:22]: Gotcha, can we edit the question to make it more clear what "this" is pointing to

Comment (Incompleteusern) [reply tex/complex-ana/quintic.tex]: I've tried to do so.

**Pull request merged**